content/plugins/environment-plugin.md

```diff
@@ -77,4 +77,22 @@ if ('development' === 'production') { // <-- default value is taken
 if ('false') { // <-- 'false' from DEBUG is taken
   console.log('Debugging output');
 }
+```
+
+## `DotenvPlugin`
+
+The third-party [`DotenvPlugin`](https://github.com/mrsteele/dotenv-webpack) (`dotenv-webpack`) allows to expose (a subset of) [dotenv variables](https://www.npmjs.com/package/dotenv):
```

Comment:
`allows to expose` sounds a bit funny...

Maybe `allows exposure of...` or `allows you to expose...`